test/e2e/smoketest.go

```diff
@@ -43,12 +43,12 @@ func SmokeTest(apiTracesEndpoint, collectorEndpoint, serviceName string, interva
 		bodyBytes, err := ioutil.ReadAll(resp.Body)
 		bodyString := string(bodyBytes)
 
-		if !strings.Contains(bodyString, "errors\":null") {
+		if (strings.Contains(bodyString, tStr)) {
+			return true, nil
+		} else if !strings.Contains(bodyString, "errors\":null") {
```

Comment:
Keep the errors check separate from the expected spas and move it above it

Reply:
@pavolloffay Are you ok with these changes?